### PR TITLE
atlas.json: update ethereum classic testnets

### PR DESCRIPTION
**Pull request by q9f**

### CODE REVIEW

File: atlas.json [hunk 1]
```diff
@@ -59,22 +59,17 @@
...
     "name": "Ubiq",
     "type": "testnet",
     "networkId": "0x2",
     "chainId": "0x9"
   },
-  {
-    "name": "Ethereum Classic",
-    "type": "testnet",
-    "networkId": "0x2",
-    "chainId": "0x3e"
```
Comment (q9f) [inline atlas.json:76]: This is the same as Morden.

File: atlas.json [hunk 2]
```diff
@@ -59,22 +59,17 @@
   },
   {
     "name": "Morden",
-    "project": "Ethereum",
+    "project": "Ethereum Classic",
     "type": "testnet",
-    "networkId": "0x2"
+    "networkId": "0x2",
+    "chainId": "0x3e"
```
Comment (q9f) [inline atlas.json:65]: Morden was adapted by Classic after the Foundation abandoned it.

File: atlas.json [hunk 3]
```diff
@@ -85,26 +80,37 @@
...
+    "networkId": "0x6",
+    "chainId": "0x6"
+  },
+  {
+    "name": "Mordor",
+    "project": "Ethereum Classic",
     "type": "testnet",
-    "networkId": "0x6"
+    "networkId": "0x7",
+    "chainId": "0x3f"
```
Comment (q9f) [inline atlas.json:113]: ref https://github.com/eth-classic/mordor

File: atlas.json [hunk 4]
```diff
@@ -85,26 +80,37 @@
...
+    "chainId": "0x5",
     "rpcUrl": "https://rpc.goerli.mudit.blog/"
   },
   {
     "name": "Kotti",
-    "project": "Ethereum",
+    "project": "Ethereum Classic",
+    "type": "testnet",
+    "networkId": "0x6",
+    "chainId": "0x6"
```
Comment (q9f) [inline atlas.json:106]: ref https://github.com/goerli/testnet#meta-data-kotti-classic

**Comment by q9f** [comment]
friendly spring bump @H34D

**Comment by H34D** [comment]
amazing, thanks a lot @q9f

**Pull request merged**